src/lithium/interestingness/ximport.py

```diff
@@ -13,31 +13,29 @@
 import os
 import sys
 
-log = logging.getLogger("lithium")  # pylint: disable=invalid-name
 
-
-def importRelativeOrAbsolute(f):  # pylint: disable=invalid-name
+def rel_or_abs_import(f):
```

Comment:
This function doesn't work except when the full path is given, which is a break from the documented behaviour.
```
$ lithium range.py /dev/null                                                                                                                                                                                Traceback (most recent call last):
  File "/home/truber/src/m/lithium/test2.7/bin/lithium", line 11, in <module>
    load_entry_point('lithium==0.1', 'console_scripts', 'lithium')()
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/reducer.py", line 1413, in main
    exit(Lithium().main())
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/reducer.py", line 1181, in main
    self.processArgs(args)
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/reducer.py", line 1333, in processArgs
    self.conditionScript = ximport.rel_or_abs_import(extra_args[0])
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/interestingness/ximport.py", line 34, in rel_or_abs_import
    module = __import__(f)
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/interestingness/range.py", line 35, in <module>
    from . import ximport
ValueError: Attempted relative import in non-package
$ lithium range /dev/null
Traceback (most recent call last):
  File "/home/truber/src/m/lithium/test2.7/bin/lithium", line 11, in <module>
    load_entry_point('lithium==0.1', 'console_scripts', 'lithium')()
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/reducer.py", line 1413, in main
    exit(Lithium().main())
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/reducer.py", line 1181, in main
    self.processArgs(args)
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/reducer.py", line 1333, in processArgs
    self.conditionScript = ximport.rel_or_abs_import(extra_args[0])
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/interestingness/ximport.py", line 34, in rel_or_abs_import
    module = __import__(f)
  File "/home/truber/src/m/lithium/test2.7/lib/python2.7/site-packages/lithium/interestingness/range.py", line 35, in <module>
    from . import ximport
ValueError: Attempted relative import in non-package
$
```

Take a look at https://github.com/jschwartzentruber/lithium/blob/total_recall/utils.py which does work in the above cases.